testing/constraints-3.6.txt

```diff
@@ -0,0 +1,11 @@
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
+# Then this file should have foo==1.14.0
+google-api-core==1.21.0
+proto-plus==1.4.0
+grpc-google-iam-v1==100000.0.0
```

Comment:
We should set a minimum version for `grpc-google-iam-v1` in setup.py. Looks like 0.9.0 is the earliest released https://pypi.org/project/grpc-google-iam-v1/#history